learningpath-api/src/main/scala/no/ndla/learningpathapi/service/ReadService.scala

```diff
@@ -40,6 +41,10 @@ trait ReadService {
 
   class ReadService {
 
+    private val getUserFeideIDMemoize = Memoize(feideApiClient.getUserFeideID)
+    def getUserFeideID(feideAccessToken: Option[FeideAccessToken]): Try[FeideID] =
+      getUserFeideIDMemoize(feideAccessToken)
+
```

Comment:
Dette trenger vel ikke være to funksjoner?
```suggestion
    private val getUserFeideID = Memoize(feideApiClient.getUserFeideID)
```